src/main/webapp/app/exercises/shared/exercise/exercise.component.ts

```diff
@@ -12,6 +12,7 @@ export abstract class ExerciseComponent implements OnInit, OnDestroy {
     private eventSubscriber: Subscription;
     @Input() embedded = false;
     @Input() course: Course;
+    @Input() isInExerciseGroup?: moment.Moment;
```

Comment:
Why was moment.Moment chosen ?

Reply:
it is a bad practice and is resolved in new push, thx :)